packages/core/src/tools/create-lambda-handler.js

```diff
@@ -205,7 +205,8 @@ const createLambdaHandler = (appRawOrPath) => {
           // Adds logging for _all_ kinds of http(s) requests, no matter the library
           if (!skipHttpPatch) {
             const httpPatch = createHttpPatch(event);
-            httpPatch(require('http')); // 'https' uses 'http' under the hood
+            httpPatch(require('http'));
+            httpPatch(require('https')); // 'https' needs to be patched separately
```

Comment:
> 'https' uses 'http' under the hood

I wonder how you found out ☝️ isn't true.

Reply:
A lot of digging and `console.log` 😅 

I saw that the patching was happening, but our proxied `request` object wasn't being called. Then I saw that `https` wasn't being patched when we patched `http`. So, apparently something changed in the `https` implementation. I scanned through some of it and it doesn't directly `require('http')` - it seems like they both `require` common modules now. 

I wasn't quite curious enough to dig back through its history to find out when that happened. But that's the exact sort of thing that is fair game for a major node version upgrade.